SharedObjects/Deck.cs

```diff
@@ -6,9 +6,51 @@
 
 namespace SharedObjects
 {
+    enum CardTypes
+    {
+        Wasabi,
+        EggNigiri,
+        SalmonNigiri,
+        SquidNigiri,
+        Sashimi,
+        Tempura,
+        Dumpling,
+        MakiRoll,
+        Pudding,
+        Chopsticks
+    }
+
     class Deck
     {
-        //Cards
+        private List<Card> Cards;
+        
+        public Deck()
+        {
+            this.ResetDeck();
+        }
+        public void ResetDeck()
+        {
+            Cards = new List<Card>();
```

Comment:
Replace this with a list of CardTypes and a switch statement instead of the if elses